azurelinuxagent/common/cgroups.py

```diff
@@ -248,9 +251,12 @@ def track_agent():
         if not CGroups.enabled():
             return
         if CGroups.is_systemd_manager():
-            CGroupsTelemetry.track_systemd_service(AGENT_NAME)
+            logger.info("Tracking systemd cgroup for {0}".format(AGENT_CGROUP_NAME))
+            CGroupsTelemetry.track_systemd_service(AGENT_CGROUP_NAME)
         else:
-            CGroupsTelemetry.track_cgroup(CGroups.for_extension(AGENT_NAME))
+            logger.info("Tracking wrapper cgroup for {0}".format(AGENT_CGROUP_NAME))
```

Comment:
I'd remove "wrapper" from the message (wrapper is the parent)

Reply:
You're right, changed.